tests/testthat/test-endpoints.R

```diff
@@ -25,3 +25,22 @@ test_that("Can construct the api", {
   expect_length(logs, 2L)
   expect_identical(logs[[1L]]$logger, "daedalus.api")
 })
+
+test_that("Can get metadata", {
+  endpoint <- daedalus_api_endpoint("GET", "/metadata")
+  res <- endpoint$run()
+  expect_true(res$validated)
+  lapply(res$data$parameters$id, function(id) {
+    expect_true(id %in% expected_parameters)
+  })
+  # expect country ids to match those from daedalus
+  country_idx <- match("country", res$data$parameters$id)
+  country_options <- res$data$parameters$options[[country_idx]]
+  daedalus_countries <- daedalus::country_names
+  expect_length(country_options, length(daedalus_countries))
+  lapply(seq_along(country_options), function(idx) {
+    daedalus_country <- scalar(daedalus_countries[[idx]])
+    expect_identical(country_options[[idx]]$id, daedalus_country)
+    expect_identical(country_options[[idx]]$label, daedalus_country)
+  })
```

Comment:
Could I suggest sort of transposing this chunk so the outermost function always returns the expectations - no sweat really as this does what's needed. If you keep the original version, I don't _think_ the `scalar()` call on `daedalus_countries` is needed.
```suggestion
  expect_identical(
    vapply(country_options, `[[`, FUN.VALUE = character(1), "id"),
    daedalus_countries
  )
  expect_identical(
    vapply(country_options, `[[`, FUN.VALUE = character(1), "label"),
    daedalus_countries
  )
```